eng/icu.proj

```diff
@@ -20,9 +41,8 @@
           Command="make -f icu.mk all TARGET_OS=$(TargetOS.ToLower()) TARGET_ARCHITECTURE=$(TargetArchitecture) ICU_TRACING=$(IcuTracing) WASI_SDK_PATH=$(WASI_SDK_PATH) $(_ExtraParams) SHELL=/bin/bash"
           IgnoreStandardErrorWarningFormat="true" />
   </Target>
-  <Target Name="Restore" />
   <Target Name="Test" />
-  <Target Name="Pack" DependsOnTargets="Build">
+  <Target Name="ReallyPack" DependsOnTargets="ReallyBuild" BeforeTargets="Pack">
```

Comment:
Why do we need `ReallyBuild` and `ReallyPack` to happen before the `Build` and `Pack` targets?

Reply:
There's some weird problem with some of the project types like Traverse, where you can't override the default targets but you CAN run your own target before or after them. Basically, if this is `Target Name="Pack"`, it just won't get executed. And we need Traverse in order to make the Restore target work.

Reply:
They're targets that prep the build and pack targets.